auth/custom_claims.js

```diff
@@ -7,58 +7,72 @@ let idToken;
 // [START set_custom_user_claims]
 // Set admin privilege on the user corresponding to uid.
 
-admin.auth().setCustomUserClaims(uid, {admin: true}).then(() => {
-  // The new custom claims will propagate to the user's ID token the
-  // next time a new one is issued.
-});
+admin
+  .auth()
+  .setCustomUserClaims(uid, {admin: true})
+  .then(() => {
+    // The new custom claims will propagate to the user's ID token the
+    // next time a new one is issued.
+  });
 // [END set_custom_user_claims]
 
-
 // [START verify_custom_claims]
 // Verify the ID token first.
-admin.auth().verifyIdToken(idToken).then((claims) => {
-  if (claims.admin === true) {
-    // Allow access to requested admin resource.
-  }
-});
+admin
+  .auth()
+  .verifyIdToken(idToken)
+  .then((claims) => {
+    if (claims.admin === true) {
+      // Allow access to requested admin resource.
+    }
+  });
 // [END verify_custom_claims]
 
 // [START read_custom_user_claims]
 // Lookup the user associated with the specified uid.
-admin.auth().getUser(uid).then((userRecord) => {
-  // The claims can be accessed on the user record.
-  console.log(userRecord.customClaims.admin);
-});
+admin
+  .auth()
```

Comment:
Is there any setting for this chaining?  The thing is that when embedded in the docs, it can look better to use less vertical space when possible. 

Generally I like to break a chain when:
  * There's a callback
  * The chain can't fit on one line

Reply:
The main thing I don't like is the breaking of the first item.

I would much prefer:
```
admin.auth()
  .foo()
  .bar()
```

To:
```
admin
  .auth()
  .foo()
  .bar()
```

The issue is that things look weird when your first variable is short:

```
db                     // <-- Look like db is just floating in space 
    .whatever()
    .foo()
```

Reply:
So I found this [rule](https://eslint.org/docs/2.0.0/rules/newline-per-chained-call)

I can make it

```javascript
"newline-per-chained-call": [
 "error",
  {
    "ignoreChainWithDepth": 4
  }
]
```

by using `"ignoreChainWithDepth": 4` it will allow:

```javascript
admin.auth().createCustomToken(uid)
  .then(function(customToken) {
    // Send token back to client
  })
  .catch(function(error) {
    console.log('Error creating custom token:', error);
  });
// [END custom
```

let me know if you agree with this change so I can either push the changes or find a better way to tackle it.

Reply:
Sorry about my slow response here!  I just did some quick experiments and that setting sounds good to me.  Thanks for finding it.